src/check.ts

```diff
@@ -319,14 +339,14 @@ See https://github.com/actions-rs/clippy-check/issues/2 for details.`);
     /// Convert parsed JSON line into the GH annotation object
     ///
     /// https://developer.github.com/v3/checks/runs/#annotations-object
-    static makeAnnotation(contents): octokit.ChecksCreateParamsOutputAnnotations {
-        const primarySpan = contents.message.spans.find((span) => span.is_primary == true);
+    static makeAnnotation(contents: AnnotationContents): octokit.ChecksCreateParamsOutputAnnotations {
+        const primarySpan: undefined | MessageSpan = contents.message.spans.find((span) => span.is_primary == true);
```

Comment:
Should not the type here be inferred automatically?

Reply:
Yeah, I'm not sure why in this case but in general typescript has trouble inferring the types of arguments I've found. You can check in the vscode editor mode by hovering over the argument. In this case it should show `contents: any` I think.